### PR TITLE
feature: adds support for images on info page

### PR DESCRIPTION
**Pull request by r1b**
**UNTESTED**

@kevin-roark I would appreciate a close review - i'm not confident that I implemented this correctly

### CODE REVIEW

**Comment by kevin-roark** [comment]
this looks good for the most part! when I can I'll modify so the imagelist works correctly here and then will merge.

File: transformer/data.js [hunk 1]
```diff
@@ -48,6 +55,12 @@ async function getYamlDatas (files) {
 
 const designerFileName = data => path.join(designerDir, `${data.slug.trim()}.yml`)
 const projectFileName = data => path.join(projectDir, `${data.slug.trim()}.yml`)
+const infoFileName = data => path.join(infoDir, `${data.slug.trim()}.yml`)
+
+async function updateInfoFile (data) {
+  // write updated designer to same file
```
Comment (r1b) [inline transformer/data.js]: oops bad comment

File: transformer/images.js [hunk 2]
```diff
@@ -15,10 +15,58 @@ const imageDir = path.join(__dirname, '../static/images')
...
+  if (processedImages.length === 0) {
+    return
+  }
+
+  const images = await Promise.all(info.photos.map(async (image, index) => {
+    const processedItem = processedImages.find(item => item.originalImageIndex === index)
+    return processedItem ? getProcessedImageData(image, processedItem) : image
+  }))
+
+  // write updated project to same file
```
Comment (r1b) [inline transformer/images.js]: oops bad comment

**Comment by r1b** [comment]
This looks fantastic - reviewed here:
https://59fe426ddf995375402abd7d--tank-commander-melody-70755.netlify.com/info

**Pull request merged**